cli/tests/integration_tests.rs

```diff
@@ -2121,6 +2121,11 @@ itest!(main_module {
   output: "main_module.ts.out",
 });
 
+itest!(net_explicit_undefined_hostname {
+  args: "run --allow-net net_explicit_undefined_hostname.js",
+  output: "net_explicit_undefined_hostname.js.out",
+});
+
```

Comment:
Can you move this to a unit test instead of an integration test?

Reply:
Fair. Did not even notice there were unit tests 😅